apps/mull-api/src/app/event/event.service.ts

```diff
@@ -92,6 +92,14 @@ export class EventService {
     return await this.eventRepository.save(event);
   }
 
+  async removeParticipant(eventId: number, userId: number) {
+    const event = await this.eventRepository.findOne(eventId, { relations: ['participants'] });
+    const user = event.participants.find((participant) => participant.id == userId);
```

Comment:
A check needs to be added here in case userId does not exist in the table since it returns `-1` if userId is not found.
Currently, if the participant DOES NOT exist but the event DOES exist, it will delete the last person that was added to the event. Essentially, it will call `event.participants.splice(-1, 1);`
If possible, ensuring that there is a test added for this would be nice!